code/modules/mob/living/carbon/examine.dm

```diff
@@ -0,0 +1,333 @@
+
+	var/msg = "<span class='info'>*---------*\nThis is "
+
+	msg += "<EM>[name]</EM>"
+
+	// Show what you are
+	msg += examine_what_am_i()
+	msg += "\n"
+
+	// All the things wielded/worn that can be reasonably described with a common template:
+	var/list/message_parts = examine_visible_clothing(skipgloves, skipsuitstorage, skipjumpsuit, skipshoes, skipmask, skipears, skipeyes, skipface)
+
+	for(var/parts in message_parts)
+		var/action = parts[1]
+		var/obj/item/item = parts[2]
+		var/preposition = parts[3]
+		var/limb_name = parts[4]
+		var/accessories = null
+		if(length(parts) >= 5)
+			accessories = parts[5]
+
+		if(item && !(item.flags & ABSTRACT))
+			var/item_words = item.name
+			if(item.blood_DNA)
+				item_words = "[item.blood_color != "#030303" ? "blood-stained" : "oil-stained"] [item_words]"
+			var/submsg = "[p_they(TRUE)] [action] [bicon(item)] \a [item_words]"
+			if(accessories)
+				submsg += " with [accessories]"
+			if(limb_name)
+				submsg += " [preposition] [p_their()] [limb_name]"
+			if(item.blood_DNA)
+				submsg = "<span class='warning'>[submsg]!</span>\n"
+			else
+				submsg = "[submsg].\n"
+			msg += submsg
+		else
+			// add any extra info on the limbs themselves
+			msg += examine_handle_individual_limb(limb_name)
+
+	//handcuffed?
+	if(handcuffed)
+		if(istype(handcuffed, /obj/item/restraints/handcuffs/cable/zipties))
+			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] [bicon(handcuffed)] restrained with zipties!</span>\n"
+		else if(istype(handcuffed, /obj/item/restraints/handcuffs/cable))
+			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] [bicon(handcuffed)] restrained with cable!</span>\n"
+		else
+			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] [bicon(handcuffed)] handcuffed!</span>\n"
+
+	//Jitters
+	switch(AmountJitter())
+		if(600 SECONDS to INFINITY)
+			msg += "<span class='warning'><b>[p_they(TRUE)] [p_are()] convulsing violently!</b></span>\n"
+		if(400 SECONDS to 600 SECONDS)
+			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] extremely jittery.</span>\n"
+		if(200 SECONDS to 400 SECONDS)
+			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] twitching ever so slightly.</span>\n"
+
+
+	var/appears_dead = FALSE
+	var/just_sleeping = FALSE //We don't appear as dead upon casual examination, just sleeping
+
+	if(stat == DEAD || HAS_TRAIT(src, TRAIT_FAKEDEATH))
+		var/obj/item/clothing/glasses/E = get_item_by_slot(slot_glasses)
+		var/are_we_in_weekend_at_bernies = E?.tint && istype(buckled, /obj/structure/chair/wheelchair) //Are we in a wheelchair with our eyes obscured?
+
+		if(isliving(user) && are_we_in_weekend_at_bernies)
+			just_sleeping = TRUE
+		else
+			appears_dead = TRUE
+
+		if(suiciding)
+			msg += "<span class='warning'>[p_they(TRUE)] appear[p_s()] to have committed suicide... there is no hope of recovery.</span>\n"
+		if(!just_sleeping)
+			msg += "<span class='deadsay'>[p_they(TRUE)] [p_are()] limp and unresponsive; there are no signs of life"
```

Comment:
```suggestion
			msg += "<span class='deadsay'>[p_they(TRUE)] [p_are()] limp and unresponsive; there are no signs of life</span>"
```

Reply:
The span gets added in later after this block is finished (since we need to add in the "their soul has departed..." bit afterwards.